extensions/kind/tsconfig.json

```diff
@@ -13,6 +17,8 @@
     ]
 },
 "include": [
-    "src"
+    "src",
+    "types/*.d.ts",
+    "../../types/**/*.d.ts"
```

Comment:
maybe this one is not required as it's outside the scope of the extension

Reply:
Removing it causes import of templates to be errored in both VSCode and WebStorm